Library/Homebrew/cmd/update-bash.sh

```diff
@@ -305,7 +323,7 @@ EOS
   wait
   trap - SIGINT
 
-  for DIR in "$HOMEBREW_REPOSITORY" "$HOMEBREW_LIBRARY"/Taps/*/*
+  for DIR in "${DIRS[@]}"
   do
     [[ -d "$DIR/.git" ]] || continue
     pull "$DIR"
```

Comment:
It won't work without the patch you mentioned above https://github.com/Homebrew/homebrew/pull/48509#discussion_r50978546